internal/form/question/service.go

```diff
@@ -0,0 +1,90 @@
+package question
+
+import (
+	"context"
+	databaseutil "github.com/NYCU-SDC/summer/pkg/database"
+	logutil "github.com/NYCU-SDC/summer/pkg/log"
+	"github.com/google/uuid"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/trace"
+	"go.uber.org/zap"
+)
+
+type Querier interface {
+	Create(ctx context.Context, params CreateParams) (Question, error)
+	Update(ctx context.Context, params UpdateParams) (Question, error)
+	Delete(ctx context.Context, params DeleteParams) error
+	ListByFormID(ctx context.Context, formID uuid.UUID) ([]Question, error)
+}
+
+type Service struct {
+	logger  *zap.Logger
+	queries Querier
+	tracer  trace.Tracer
+}
+
+func NewService(logger *zap.Logger, db DBTX) *Service {
+	return &Service{
+		logger:  logger,
+		queries: New(db),
+		tracer:  otel.Tracer("forms/service"),
+	}
+}
+
+func (s *Service) Create(ctx context.Context, input CreateParams) (Question, error) {
+	ctx, span := s.tracer.Start(ctx, "CreateQuestion")
+	defer span.End()
+	logger := logutil.WithContext(ctx, s.logger)
+
+	q, err := s.queries.Create(ctx, input)
+	if err != nil {
+		err = databaseutil.WrapDBError(err, logger, "create question")
+		span.RecordError(err)
+		return Question{}, err
+	}
+	return q, nil
+}
+
+func (s *Service) Update(ctx context.Context, input UpdateParams) (Question, error) {
+	ctx, span := s.tracer.Start(ctx, "UpdateQuestion")
```

Comment:
there's no need to repeat "Question" in the span name here, please update the rest of the code accordingly